app/elements/youneeq-tracker/youneeq-tracker.js

```diff
@@ -134,7 +134,7 @@ class youneeqTracker {
   }
 
   _contentChanged(content, oldContent) {
-    if (typeof content !== 'undefined' && Object.keys(content).length > 0) {
+    if (typeof content !== 'undefined' && Object.keys(content).length > 0 && !this.hidden) {
```

Comment:
So, essentially that would be within the content itself, so cranberry-story and cranberry-gallery elements. I must have mislead or misspoke when I referred to that, or was thinking that you were going to attempt to do something different with the tracker / content items.

That said, there should be some checks within youneeq-content as it will still receive the "youneeq-callbackid" and the "youneeq-suggestions" events even if the youneeq-content element is hidden. If you want I can kind of show you what I mean by modifying your branch a little bit and then you can tell me if that is what you were going for.